index.html

```diff
@@ -1401,9 +1401,47 @@ <h2>Core Vocabulary Definitions</h2>
 <p>
      <span class="rfc2119-assertion" id="td-context-default-language">
      In the <code>@context</code> array, the optional default language that is 
-     used throughout a TD document is defined within an object as a member with 
+     used throughout a TD document for human-readable strings outside of a <a href="#multilanguage">MultiLanguage</a> object
+     is also defined within an object as a member with 
      a key <code>@language</code> whose value MUST be a string representing a [[!BCP47]] 
      language code.</span>
+     <span class="rfc2119-assertion" id="td-context-default-language-last">
+     The object used to define the default language MUST be the last object in
+     the `@context` array.
+     </span>
+     <span class="rfc2119-assertion" id="td-context-default-language-unique">
+     There MUST be only one object defining a default language in an `@context` array.
+     </span>
+     <span class="rfc2119-assertion" id="td-context-default-language-separate">
+     The object used to define the default language MUST NOT be combined with any other
+     objects defining context extensions (additional vocabulary terms).
```

Comment:
This is an issue when a JSON-LD library is used to serialize a TD. I would rather relax this. (Also applies to the position in the array...)

Reply:
Hmmm.... good point, I guess.   And to the other point.   Well, I thought I would throw this out there, I'm not married to it.

Reply:
Lucky you :D
Good remove it.

Reply:
Removed.   I also removed the assertion that was an aim to avoid duplicates.   However, this is a serialization issue and should be handled by JSON-LD.